### ISSUE TITLE
Error with Nuxt 1.0.0-rc2

### ISSUE THREAD

**Issue by vladzur** [nuxt-community/starter-template#24]
I'm having an error message after running a fresh installation using Nuxt/starter
```
ERROR  Failed to compile with 1 errors                                                                                              23:08:12

This dependency was not found:

* ~components/Logo.vue in ./node_modules/babel-loader/lib?{"presets":["vue-app"],"babelrc":false,"cacheDirectory":true}!./node_modules/vue-loader/lib/selector.js?type=script&index=0!./pages/index.vue
```
I have another project It was running fine until I ran yarn upgrade
Nuxt 1.0.0-rc2 causes the same error

Full output:
```
MisProyectos|⇒ vue init nuxt-community/starter-template prueba-nuxt

? Project name prueba-nuxt
? Project description Nuxt.js project
? Author vladzur <vladzur@gmail.com>

   vue-cli · Generated "prueba-nuxt".

   To get started:
   
     cd prueba-nuxt
     npm install # Or yarn
     npm run dev

MisProyectos|⇒ cd prueba-nuxt 
prueba-nuxt|⇒ yarn
yarn install v0.27.5
info No lockfile found.
[1/4] Resolving packages...
[2/4] Fetching packages...
warning fsevents@1.1.2: The platform "linux" is incompatible with this module.
info "fsevents@1.1.2" is an optional dependency and failed compatibility check. Excluding it from installation.
[3/4] Linking dependencies...
[4/4] Building fresh packages...
success Saved lockfile.
Done in 68.41s.
prueba-nuxt|⇒ yarn run dev
yarn run v0.27.5
$ nuxt
  nuxt:build App root: /home/vladzur/MisProyectos/prueba-nuxt +0ms
  nuxt:build Generating /home/vladzur/MisProyectos/prueba-nuxt/.nuxt files... +3ms
  nuxt:build Generating files... +10ms
  nuxt:build Generating routes... +33ms
  nuxt:build Building files... +49ms
  nuxt:build Adding webpack middleware... +278ms
Build completed in 9.36s



 ERROR  Failed to compile with 1 errors                                                                                              23:44:10

This dependency was not found:

* ~components/Logo.vue in ./node_modules/babel-loader/lib?{"presets":["vue-app"],"babelrc":false,"cacheDirectory":true}!./node_modules/vue-loader/lib/selector.js?type=script&index=0!./pages/index.vue

To install it, you can run: npm install --save ~components/Logo.vue
```

<div align="right"><sub><em>This question is available on <a href="https://nuxtjs.cmty.io">Nuxt.js</a> community (<a href="https://nuxtjs.cmty.io/nuxt-community/starter-template/issues/c11">#c11</a>)</em></sub></div>

**Comment by sewede** [comment]
In your pages/index.vue file change the import statement to:

`import Logo from '~/components/Logo.vue'`

Note the slash.

The latest version of nuxt seems to have a problem with the alias "~" without the slash. 

**Update:** This issue is fixed with #23

**Comment by vladzur** [comment]
Ok, thanks a lot!
